tests/cephadm/test_cephadm_ansible_manage_services.py

```diff
@@ -0,0 +1,49 @@
+from os.path import basename
```

Comment:
This looks similar to [1], try to utilize same test module.
[1] https://github.com/red-hat-storage/cephci/pull/2049